### DISCUSSION TITLE
DatePickerInput testing onChange

### DISCUSSION THREAD

**Discussion by oiqwrwer1** [mantinedev/mantine#3751] [Q&A]
https://github.com/mantinedev/mantine/blob/0b3744c37832f53b95abfa3cf1426d8cd411719f/src/mantine-dates/src/components/DatePickerInput/DatePickerInput.test.tsx#L23

It would be helpful for the test to have tests showing how to test the onChange of the DatePickerInput component.
B/c of the changes in Mantine v6, I have to change my use of DatePicker to DatePickerInput. The Change Log suggests the change is in name only, but the internals changed, so my existing unit tests for firing onChange events to the DatePickerInput no longer actually trigger the onChange.

**Comment by rtivital** [comment]
You can find these tests here – https://github.com/mantinedev/mantine/blob/master/src/mantine-dates/src/tests/it-supports-date-input-props.tsx#L1
